### DISCUSSION TITLE
Crazyradio flashing firmware

### DISCUSSION THREAD

**Discussion by Richgithub11** [bitcraze/discussions#359] [Q&A] [answered]
Hi,

I am having issues at the 'make install' stage of flashing the firmware. I have tried on various flavors of linux and various instructions - all with an error at this stage. I am guessing the hardware is a little old and nobody has written instructions for it in a long time. Before I send this back to where it came, is there any up-to -date instructions to use so I can flash the firmware and use Mousejack//Jackit etc?

Any help very much appreciated!

**Comment by ataffanel** [comment, marked as answer]
What firmware are you trying to flash? The Crazyradio firmware or another one?

For the Crazyradio firmware, the documented procedure should still work (I have updated the python script for python3 a while ago to keep it alive, it you have any problem with it please make a ticket in the firmware repos): https://github.com/bitcraze/crazyradio-firmware#flashing-the-firmware. For the other firmwares, I suggest you ask the question on the corresponding git repos/forum. We do not have much experience with other firmwares on the Crazyradio.

**Comment by Richgithub11** [comment]
Thanks for the help. I was able to get your tutorial to work properly. I was unable to get the mousejack stuff to work since I guess it isn't supported any more. Not sure why the hardware and out-of-date tutorials exist. Waste of time and money.

**Reply by ataffanel** [reply]
The Crazyradio dongle is designed to communicate and control the Crazyflie quadcopter and other Bitcraze product. Being open-source it has also been used by other projects like mousejack.

This is not something we will be able to help you with in this forum, but you should ask on the mousejack, or related projects, github or community: those project are also open-source and the hardware has not changed so I see no reason why anything would stop working.

**Reply by Richgithub11** [reply]
I appreciate your reply very much. I also appreciate the help. I will go bother others now lol! Thanks again.
